spec/models/note_spec.rb

```diff
@@ -10,4 +12,17 @@
   it 'encrypts the body' do
     expect(note.body_before_type_cast).not_to eq('This is the body')
   end
+
+  context 'when the body has content' do
+    it 'filled is true' do
```

Comment:
What's the difference in `rspec -f doc` if this spec were to just use `specify { ... }`

Reply:
So `specify` is essentially the same as `it`? But I suppose the advantage here of `specify` is that it would read more like a sentence
[ed74e84](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/645/commits/ed74e848a17c6512d20303d7c8683d5dae245848)

Reply:
I mean what's the rspec doc output if you use specify without a param.

```ruby
specify { expect(note).to be_filled }
```

Reply:
Ah I see. I didn't know you could write it like that. Nice readable output:
```
Note
    when the body has content
        is expected to be filled
```

Reply:
[5b577a6](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/645/commits/5b577a65d239f0746f88bcd1e280a0b4dbb38a04)